src/services/ghost/strategies/CommentHelpers.ts

```diff
@@ -38,56 +38,6 @@ export function isCommentLine(line: string, languageId: string): boolean {
 	return false
 }
 
-export function extractComment(document: TextDocument, currentLine: number): string {
```

Comment:
unrelated, but when i removed use of another import of autotrigger strategy i noticed this was unused as well